### DISCUSSION TITLE
Do you have a mouthpiece recommendation?

### DISCUSSION THREAD

**Discussion by RyanParsley** [cardonabits/haxo-hw#25] [General]
I don't know the first thing about a saxophone, but this project looks like a fun excuse to remedy that. Any recommendations on an appropriate mouthpiece for this project?

**Comment by jcard0na** [comment]
You do not need a mouthpiece to get started and confirm that you've built a functional instrument:  blowing air through the neck holes is sufficient.  After that, just pick any mouthpiece that feels comfortable to you.  If you intend to play the tenor, I'd get a tenor mouthpiece so you start getting used to the larger size.  But do not buy expensive mouthpieces!  Contrary to a real saxophone, the mouthpiece you choose will have no effect on sound quality on the haxophone.

**Comment by stardusthamster** [comment]
One more question concerning the mouthpiece: 
A friend of mine has an alto sax and we tried the mouthpiece on the haxophone. It would not fit, because it was too small.
Are alto mouthpieces not suited? Are they too small?

Thank you for helping a saxo newbie.

**Reply by jcard0na** [reply]
Hi,

The neck is 16mm in diameter, which works with some alto mouthpieces and all tenor mouthpieces, if you add a bit of washi tape for a snug fit.

![image](https://github.com/cardonabits/haxo-hw/assets/676181/1bf4aeff-e7df-4ed4-b51d-462dde97ceee)

If you have a specific mouthpiece you want to use that does not fit, you can always modify and print [another neck](https://github.com/cardonabits/haxo-hw/blob/main/models/neck-neck.stl)!

Cheers,

**Reply by stardusthamster** [reply]
Thank you, good idea!
